govuk-paas/paas-rds-broker-boshrelease
# [#118484173] Remove unused jobs from release.

# What

In order to reduce the maintenance burden, we want to strip out parts of the boshrelease that aren't being used.  This removes several errand jobs that we aren't using, and don't intend to use.
# How to test

Deploy the pipeline using the aws-broker release from this branch as follows:
- Edit the `aws-broker-boshrelease` resource in the pipeline, and replace the `tag_filter` with `branch: remove_unused_jobs` (line 98).
- Edit `manifests/cf-manifest/deployments/060-aws-broker.yml` and change the release version to something else (it can be anything that doesn't match a version that's been previously uploaded to the bosh director)

Commit these changes to a branch, and run the pipeline using this branch.  Verify that the custom-acceptance-tests pass using this new release.
# Who can review

Anyone but @HenryTK or myself.
